src/fides/api/service/messaging/message_dispatch_service.py

```diff
@@ -341,6 +343,7 @@ def _build_email(  # pylint: disable=too-many-return-statements
             body=_render(messaging_template.content["body"]),  # type: ignore
         )
     if action_type == MessagingActionType.PRIVACY_REQUEST_ERROR_NOTIFICATION:
+        # get email template doesn't use defualt templates in messaging_template.py
```

Comment:
```suggestion
        # get email template doesn't use default templates in messaging_template.py
```

Reply:
Can you further explain this code comment, I don't understand what you mean here

Reply:
Oh, some action types, like `MessagingActionType.PRIVACY_REQUEST_ERROR_NOTIFICATION` for example, don't use the default templates that are configurable in the Admin-UI. They are instead hard-coded in `fides`, which we retrieve using `get_email_template(action_type)`

Reply:
thank you I didn't realize that distinction! do you mind adding some of this detail to the code comment, and fixing the typo while you're there

Reply:
absolutely!